spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/sql/GcpCloudSqlAutoConfiguration.java

```diff
@@ -56,7 +55,6 @@
 @Import({AppEngineJdbcInfoProviderAutoConfiguration.class,
 		MySqlJdbcInfoProviderAutoConfiguration.class,
 		PostgreSqlJdbcInfoProviderAutoConfiguration.class})
-@ConditionalOnBean(CloudSqlJdbcInfoProvider.class)
```

Comment:
How about if move this to the `cloudSqlDataSourceProperties` `@Bean` level?

Reply:
That works! I should've thought of that...